guides/v2.2/cloud/configure/setup-cron-jobs.md

```diff
@@ -104,29 +104,41 @@ The [auto-crons feature](#verify-cron-configuration) must be enabled on your {{s
 
 #### To add custom crons
 
-1. In your local development environment, edit the `.magento.app.yaml` file in the Magento `/app` directory.
+1. In your local development environment, edit the `.magento.app.yaml` file in the Magento `/app` directory. 
+
+1. The structure of `crons` section in the `.magento.app.yaml` is:
```

Comment:
This does not qualify as a step, because there is no action. I have a recommendation for placement of the sample code in the next step.

Reply:
Thank you, for the explanation!